app/src/main/java/edu/grinnell/kdic/MainActivity.java

```diff
@@ -374,13 +380,13 @@ public void showVisualizeFragment() {
             mPlaybackToolbar.setNavigationIcon(R.drawable.ic_keyboard_arrow_down_white_24dp);
 
             // move the play button to the middle
-            final float shiftAmnt = (mPlaybackToolbar.getWidth() - playPauseButton.getWidth()) / 2;
+            final float shiftAmnt = (mPlaybackToolbar.getWidth() - mPlayPauseButton.getWidth()) / 2;
             TranslateAnimation animation = new TranslateAnimation(shiftAmnt, 0, 0, 0);
             animation.setDuration(200);
             animation.setInterpolator(new AccelerateDecelerateInterpolator());
             animation.setFillAfter(true);
-            playPauseButton.setTranslationX(-1 * shiftAmnt);
-            playPauseButton.startAnimation(animation);
+            mPlayPauseButton.setTranslationX(-1 * shiftAmnt);
+            mPlayPauseButton.startAnimation(animation);
 
             // move the info off the screen
             AlphaAnimation alphaAnimation = new AlphaAnimation(1f, 0f);
```

Comment:
What is 1f? A magic number ? Please help us make more meaning of these by extracting them as constants and naming them properly.